server/index.js

```diff
@@ -37,7 +38,8 @@ server.connection({
 
 server.register([
   HapiAuthCookie,
-  Bell
+  Bell,
+  Lout
```

Comment:
I think it's fine to include it, and probably fine to load the lout route in production, too: our code's open source, we aren't really exposing anything.